packages/framework/src/types/validator.types.ts

```diff
@@ -6,6 +6,7 @@ export type ValidateFunction<T = unknown> = AjvValidateFunction<T> | ((data: T)
 
 export type ValidationError = {
   path: string;
+  property: string;
```

Comment:
The mapping of the validation error to the schema object uses the property attribute with  `.`  and not `/`